pkg/report/report.go

```diff
@@ -130,12 +131,16 @@ func GetSerializer(format v1alpha1.ReportFormatType) (ReportSerializer, error) {
 	}
 }
 
-func (report *TestsReport) SaveReportBasedOnType(reportFormat v1alpha1.ReportFormatType, reportName string) error {
+func (report *TestsReport) SaveReportBasedOnType(reportFormat v1alpha1.ReportFormatType, reportPath, reportName string) error {
 	serializer, err := GetSerializer(reportFormat)
 	if err != nil {
 		return err
 	}
-	filePath := reportName + "." + strings.ToLower(string(reportFormat))
+	fileName := reportName + "." + strings.ToLower(string(reportFormat))
```

Comment:
should we make sure that repotName is just the name not the path ?

Reply:
I think `filepath.Join` will still work fine

Reply:
not sure on this

Reply:
https://go.dev/play/p/ZpeeEvjNwKR